src/ansys/templates/python/pyansys_advanced/{{cookiecutter.__project_name_slug}}/README.rst

```diff
@@ -79,7 +79,7 @@ need to follow these steps:
 
     .. code:: bash
 
-        git clone {{ cookiecutter.repository_url }}
+      git clone {{ cookiecutter.repository_url }}
 
```

Comment:
I am guessing you will need to change more lines right? =)

Reply:
Sure!

But I like to have the PR opened first (for the screenshots purposes and such)